src/main/java/com/uid2/shared/attest/AttestationTokenRetriever.java

```diff
@@ -130,9 +129,12 @@ private void attestationExpirationCheck(long timerId) {
             }
 
             attest();
-        } catch (AttestationTokenRetrieverException | IOException e) {
+        } catch (AttestationTokenRetrieverException e) {
             notifyResponseWatcher(401, e.getMessage());
             LOGGER.info("Re-attest failed: ", e);
+        } catch (IOException e) {
```

Comment:
Sorry to do this as I know you have been trying to get this approved for a while, but can we log out for both the 401 and the 500 the response code, and change the messages so we can tell what the cause was?

Reply:
I am not quite sure what you are suggesting. It will already log out for both a 401 and a 500, with the message being based on the exception that occurred. Are you suggesting we generalize the error that is logged, so that a 401 always logs the same thing and a 500 always logs the same thing? And if so what do you suggest for the message?

Reply:
What I want to make sure is that we can tell the difference in our logs (so we can alert differently) between the 2 types of 401 errors. If you are confident that this is the case, then I am OK with this.